### PR TITLE
half/double view range instead of adding/removing 10 to it

### PR DESCRIPTION
**Pull request by HybridDog**
see https://github.com/minetest/minetest/pull/3404

### CODE REVIEW

**Comment by proller** [comment]
Maybe  change 2 ->  1.5  ?


Also controlling farmesh on R pressing can be very useful.

**Comment by HybridDog** [comment]
>Maybe change 2 -> 1.5 ?

done
>Also controlling farmesh on R pressing can be very useful.

What do you mean?

**Comment by donat-b** [comment]
> Also controlling farmesh on R pressing can be very useful.

Quite nice idea, but maybe it's not always handy. Sometimes I really want to see really far.
Maybe we need a separate mapping for that?

**Comment by HybridDog** [comment]
>Sometimes I really want to see really far.

You could add a setting to enable local map loading, on servers you then wouldn't need to wait until the chunk is sent to you.

File: games/default [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit a266a308649e890263fd7376f89d49ad7dac1d36
+Subproject commit 8f73df4f75b023c5350c7da8bbe3dbead0e6381d
```
Comment (proller) [inline games/default]: please dont commit this

File: games/pixture [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit f602cb022fef8e76b86bae78f68c60e94ca87973
+Subproject commit 96130c3407f9c40e7ccd3d706fb20f4994440a89
```
Comment (proller) [inline games/pixture]: and this

Comment (HybridDog) [inline games/pixture]: How can l remove them from the commit?

**Comment by proller** [comment]
git reset upstream/master 
and commit again,
then push -f

**Comment by HybridDog** [comment]
>and commit again,

This would mean l need to do the changes again, don't l?
(l would copy the file, remove the latest commit using rebase -i, replace the file and commit again)
Is there another way?

**Comment by donat-b** [comment]
That happened because you didn't update the submodules. I *suppose* you could do something like this:
`git reset HEAD~1 --soft`
`git submodule update --init --recursive`
Then commit again and push -f

**Pull request merged**